datajoint/condition.py

```diff
@@ -61,6 +63,18 @@ def append(self, restriction):
             super().append(restriction)
 
 
+@dataclass
+class Top:
+    """
+    A "restriction" to set the sorting clauses of a query. Since it is not a true
+    restriction, it has no effect on the WHERE clause.
```

Comment:
```suggestion
    A restriction to the top entities of a query. In SQL, this corresponds to ORDER BY ... LIMIT ... OFFSET
```

Reply:
restricting by `dj.Top` meets DataJoint's definiton of a restriction, so it's a true restriction. The fact that SQL does this differently does not make this family of restrictions any less true.